basic-features.md

```diff
@@ -1,80 +1,149 @@
+| [Database migration toolkit (DM)](/migration-overview.md)                                                | Y            | Y            | Y            | Y            |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                                      | Deprecated   | Deprecated   | Deprecated   | Deprecated   |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                    | Y            | Y            | Y            | Y            |
+| **Management, observability and tools**                                                                  | **5.2**      | **5.1**      | **5.0**      | **4.0**      |
+| [TiDB Dashboard](/dashboard/dashboard-intro.md)                                                          | Y            | Y            | Y            | Y            |
+| [SQL diagnostics](/information-schema/information-schema-sql-diagnostics.md)                             | Experimental | Experimental | Experimental | Experimental |
+| [Information schema](/information-schema/information-schema.md)                                          | Y            | Y            | Y            | Y            |
+| [Metrics schema](/metrics-schema.md)                                                                     | Y            | Y            | Y            | Y            |
+| [Statements summary tables](/statement-summary-tables.md)                                                | Y            | Y            | Y            | Y            |
+| [Slow query log](/identify-slow-queries.md)                                                              | Y            | Y            | Y            | Y            |
+| [TiUP deployment](/tiup/tiup-overview.md)                                                                | Y            | Y            | Y            | Y            |
+| Ansible deployment                                                                                       | N            | N            | N            | Deprecated   |
+| [Kubernetes operator](https://docs.pingcap.com/tidb-in-kubernetes/)                                      | Y            | Y            | Y            | Y            |
+| [Built-in physical backup](/br/backup-and-restore-use-cases.md)                                          | Y            | Y            | Y            | Y            |
+| Top SQL                                                                                                  | Y            | N            | N            | N            |
+| [Global Kill](/sql-statements/sql-statement-kill.md)                                                     | Experimental | Experimental | Experimental | Experimental |
+| [Lock View](/information-schema/information-schema-data-lock-waits.md)                                   | Experimental | Experimental | Experimental | Experimental |
```

Comment:
Please update this column since 'Lock View' has been GAed: https://docs.pingcap.com/tidb/stable/release-5.2.0